src/client/components/pages/parts/revisions-table.js

```diff
@@ -116,9 +117,19 @@ function RevisionsTable(props) {
 														revision.notes.map(note => (
 															<div className="revision-note clearfix" key={note.id}>
 																<span className="note-content">
-																	{note.content}
+																	<span
+																		// eslint-disable-next-line react/no-danger
+																		dangerouslySetInnerHTML={{
+																			__html: sanitize(note.content.replace(
+																				// eslint-disable-next-line max-len
+																				/((http|https|ftp|ftps)\:\/\/[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,3}(\/\S*)?)/g,
```

Comment:
Regarding the url extension part, `.[a-zA-Z]{2,3}` is too restrictive. For example `http://www.test.info/` will return `http://www.test.inf` with a missing final 'o'.
I also wonder if we should also convert `www.test.info` without the protocol at the start. What do you think?

A quick google search an i found this which seems to solve both issues:
https://www.urlregex.com/